tests/tools/interpreter_test.py

```diff
@@ -0,0 +1,16 @@
+from ...tools.interpreter import Interpreter
+import logging
+import unittest
+from ...tests.kernel.wave.wave_gemm_test import Test as wave_gemm_test
+
+
+class Test(unittest.TestCase):
```

Comment:
nti: maybe switch to pytest entirely? You won't need this `unittest.main()` as well

Reply:
sure will remove.